### ISSUE TITLE
executable path

### ISSUE THREAD

**Issue by mroswell** [kobotoolbox/xls-import#3]
I merged #2 (thanks!) -- but given this:
```
$ which python
/Users/marjorieroswell/anaconda2/bin/python
```
I'm going to need to figure out how to adjust the path (or set up a virtual environment?) so that the executable functions. For now I'll continue running with the python command.

**Comment by jnm** [comment]
Right, but `/usr/bin/env python` is supposed to find the absolute path of `python` in your environment. This wouldn't work for Windows, but it should work on Mac (I tried it in Sierra).

**Comment by mroswell** [comment]
If I don't start with python, I get:
```
-bash: xls2xml: command not found
```
So, created an issue for when I have a moment to figure out why. 
(I'm also on Sierra.)

I also get:
```
$ which env
/usr/bin/env
```
Anyhow, minor issue; at least it works for others.

**Comment by jnm** [comment]
Good that your `which env` output matches the `/usr/bin/env` path that I used. If you're trying `xls2xml` and it's not working, try `./xls2xml`. There's actually a UNIX security justification for having to do this: https://unix.stackexchange.com/a/4459
